roles/v2/test-utils/pool/Cargo.toml

```diff
@@ -2,17 +2,20 @@
 name = "test-pool"
 version = "0.1.0"
 edition = "2018"
-
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-codec_sv2 = { path = "../../../../protocols/v2/codec-sv2", features=["noise_sv2"] }
-roles_logic_sv2 = { path = "../../../../protocols/v2/roles-logic-sv2" }
-const_sv2 = { path = "../../../../protocols/v2/const-sv2" }
+anyhow = { path = "../../../../utils/anyhow" }
```

Comment:
handle errors in a new file `roles/v2/pool/src/error.rs` with an `enum Error` format